Content.Server/Emp/EmpSystem.cs

```diff
@@ -31,21 +31,26 @@ public void EmpPulse(MapCoordinates coordinates, float range, float energyConsum
     {
         foreach (var uid in _lookup.GetEntitiesInRange(coordinates, range))
         {
-            var ev = new EmpPulseEvent(energyConsumption, false, false);
-            RaiseLocalEvent(uid, ref ev);
-            if (ev.Affected)
-            {
-                Spawn(EmpDisabledEffectPrototype, Transform(uid).Coordinates);
-            }
-            if (ev.Disabled)
-            {
-                var disabled = EnsureComp<EmpDisabledComponent>(uid);
-                disabled.DisabledUntil = Timing.CurTime + TimeSpan.FromSeconds(duration);
-            }
+            DoEmp(uid, energyConsumption, duration);
         }
         Spawn(EmpPulseEffectPrototype, coordinates);
     }
 
+    public void DoEmp(EntityUid uid, float energyConsumption, float duration)
```

Comment:
probably not a good idea to separate into a new method since a) this does not prevent any duplication and b) this doesnt really "do" a proper EMP since it doesnt spawn a `EmpPulseEffectPrototype` like EmpPulse does

Reply:
You can suggest a different name of course, but I was shocked there wasn't a way to apply the effects of an EMP onto a single entity, and it actively impeded something I was working on. 

I feel separating am AoE EMP burst from the actual EMP inflicting an entity is perfectly reasonable, especially since *all* the method does is AoE the effects onto all the entities, and spawn the visual effect. No way around that is silly.

Reply:
Yeah then rename it and add xmldocs to it

Reply:
Will do. Suggestions for the name? ApplyEmp? EntityEmp? EmpDisable? InflictEmp?

Reply:
We normally follow a pattern of having a strict method called `TrySomeFeature` that does some sanity checks and then calls another method `DoSomeFeature` that does/forces the thing without or with very little checks

Reply:
This seem fine to you?